Cleaned up if/else branches

anmousyon opened the pull request:
More readable code still functions the same

anmousyon committed:
cleaned up if, elif, else statements

Much more readable code than before, adds two lines

anmousyon committed:
Merge pull request #1 from anmousyon/anmousyon-patch-1

cleaned up if, elif, else statements

spladug commented on reddit_liveupdate/utils.py:
```diff
@@ -10,30 +10,23 @@
+    return format_datetime(
+        datetime=dt,
+        tzinfo=pytz.UTC,
+        date_format="dd MMM YYYY HH:mm",
```
shouldn't this be referencing the new variable?

anmousyon replied:
Oh yes, sorry. Had to refork and copy paste some stuff. Must've gotten lost in that. I'll fix that really quick.

spladug commented:
[:nail_care:](https://github.com/reddit/reddit/wiki/The-Code-Salon)

Thanks for your contribution!

anmousyon committed:
Fixed line on suggestion

anmousyon commented:
![image](https://cloud.githubusercontent.com/assets/12771363/16031176/f0f74262-31bd-11e6-8fd3-ec89d6a2ff7a.png)

anmousyon committed:
More obvious build_toolbars is returning a list.

Easier to read, functions the same.
More consistent with similar function in class below.

anmousyon committed:
changed extend to append

Only adding a single element, so append makes more sense.
More consistent with similar function in class below.
